src/replicator.js

```diff
@@ -89,7 +92,11 @@ class Replicator {
     this._pubsub = new Pubsub(this.ipfs, (await this.ipfs.id()).id)
     // Identity not used, passes ref to 3ID orbit identity provider
     const identity = await Identities.createIdentity({ id: 'nullid' })
-    this._orbitdb = await OrbitDB.createInstance(this.ipfs, { directory: opts.orbitPath, identity })
+    const cachePath = path.join(opts.orbitPath || './orbitdb' , '/cache')
+    const levelDown = Storage(null, {})
+    const cacheStorage = await levelDown.createStore(cachePath)
+    const cache = new Cache(cacheStorage)
```

Comment:
Would you mind leaving a comment on what this does? (for later reference)

Reply:
👍